common/djangoapps/third_party_auth/models.py

```diff
@@ -500,6 +500,7 @@ class LTIProviderConfig(ProviderConfig):
     )
 
     lti_hostname = models.CharField(
+        default='localhost',
```

Comment:
It's kind of tricky because this field is/was required, and the migrations required that there be a default value for filling in older columns, but I don't think a default value makes sense for this field. I don't mind it if that's what needs to happen for the Django 1.8 upgrade, though.